e2e-tests/dev.env

```diff
@@ -1,6 +1,8 @@
 PM_HOST="https://api.dev.platform.pagopa.it"
+NPG_WALLET_URL="http://localhost/npg-onboarding"
```

Comment:
You have to configure the reference to dev and uat in the different env files

Reply:
I've removed this variable since the redirectUrl is now retrieved from the api service.